packages/manager/modules/exchange/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -413,12 +413,23 @@
   "exchange_tab_domain_diagnostic_dkim_title_activation": "Activer DKIM",
   "exchange_tab_domain_diagnostic_dkim_activate": "Activer",
   "exchange_tab_domain_diagnostic_dkim_deactivate": "Désactiver",
+  "exchange_tab_domain_diagnostic_dkim_title_configuration": "Configuration DKIM",
   "exchange_tab_domain_diagnostic_dkim_configuration_nok": "Votre configuration est incorrecte, veuillez suivre les indications du <a href=\"{{url}}\" target=\"_blank\">guide</a>.",
-  "exchange_tab_domain_diagnostic_dkim_activation_no_ovhcloud": "Votre domaine n'étant pas hébergé chez OVHCloud nous ne pouvons pas le configurer automatiquement. Vous pouvez configurer DKIM en suivant ce <a href=\"{{url}}\" target=\"_blank\">guide</a>.",
-  "exchange_tab_domain_diagnostic_dkim_deactivation": "DKIM est actuellement actif sur votre domaine. Souhaitez-vous le desactiver ? (non recommandé)",
+  "exchange_tab_domain_diagnostic_dkim_activation_no_ovhcloud": "Votre domaine n'étant pas hébergé chez OVHCloud nous ne pouvons pas le configurer automatiquement. Souhaitez-vous débuter la configuration ?",
+  "exchange_tab_domain_diagnostic_dkim_configurate_no_ovhcloud": "Configurer",
+  "exchange_tab_domain_diagnostic_dkim_next_no_ovhcloud": "Suivant",
+  "exchange_tab_domain_diagnostic_dkim_confirm_no_ovhcloud": "Fermer",
+  "exchange_tab_domain_diagnostic_dkim_id_generation_no_ovhcloud": "La génération de vos identifiants sont en cours, veuillez patienter...",
```

Comment:
I think there is a mistake, in my opinion it's the "génération" that is "en cours" and it should be "est en cours".

Reply:
Same as emailPro, I will change it to "est en cours" ;)